src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -309,6 +310,28 @@ def minimize_corpus(self, target_path, arguments, input_dirs, output_dir,
     """
     raise NotImplementedError
 
+  def _get_smallest_crasher(self, workdir_path):
```

Comment:
How long is this test btw?

Reply:
This new test takes ~0.17s.
All 25 tests take ~22.162s in total, the slowest two are `test_crash_timeout`s from `UnshareIntegrationTest` and `IntegrationTest`, taking ~8.09s each.